pyproject.toml

```diff
@@ -23,18 +23,18 @@ classifiers = [
 ]
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = "^3.6.2"
 protobuf = "^3.12"
 grpcio = "^1.30"
-betterproto = { version = "^2.0.0b1", allow-prereleases = true }
+betterproto = { version = "^2.0.0b4", allow-prereleases = true }
```

Comment:
Does this require a regeneration of the python code?

Reply:
Good point. I actually still have issues executing `make update` on macOS. Does it still work for you? I'll try to setup a `Dockerfile` to get a reproducible / consistent requirement where we could run `make update` on different systems.  dropped this change for now, so that this PR is only about updating the other dependencies as it was before.

Reply:
Got it working locally. 2.0.0b4 does not work yet with the gnmi proto, since there is an open bug: https://github.com/danielgtaylor/python-betterproto/issues/312